commit 4b3e2e9849d9f4c4d1a1640ce4130e305dada0e8
Author: Vishesh Handa <me@vhanda.in>
Date:   Tue Jun 21 09:32:57 2016 +0000

    Remove backtrace config files
    
    We do not care about backtrace, and this is a quick hack to make sure it
    is not included.
    
    Ideally, Akonadi should not try to include it if it is not found, since
    it is optional. But oh well!

diff --git a/config-akonadi.h.cmake b/config-akonadi.h.cmake
index 37223f7..eed0df8 100644
--- a/config-akonadi.h.cmake
+++ b/config-akonadi.h.cmake
@@ -1,8 +1,3 @@
-#cmakedefine01 Backtrace_FOUND
-#if Backtrace_FOUND
-# include <@Backtrace_HEADER@>
-#endif
-
 #cmakedefine HAVE_UNISTD_H 1
 #cmakedefine HAVE_SOPRANO 1
 
